### PR TITLE
Add doctrine migration example

### PR DESCRIPTION
**Pull request by alexander-schranz**
Add missing migration. 

https://github.com/sulu/SuluContentBundle/blob/enhancement/mapper-collection-parameter/UPGRADE.md

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -33,7 +33,117 @@ ALTER TABLE test_example_dimension_contents DROP dimension_id;
 DROP TABLE cn_dimensions;
 ```
 
-TODO provide here a general doctrine migration which support up/down.
+If you are using the `DoctrineMigrationBundle` you maybe want a `up` and `down`
+migration for your entity to make this easier you can generate the migration. Copy from
+there the tableName, foreignKey and indexName and replace the up and down methods with
+the one of the following example:
```
Comment (niklasnatter) [inline UPGRADE.md]: not sure what you mean with `Copy from there the tableName, foreignKey and indexName`? 🤔 

```suggestion
If you are using the `DoctrineMigrationBundle` you can also reuse the following migration class
to migrate the data of you entity. Make sure to provide the correct `tableName`, `foreignKey` 
and `indexName` before executing the migration.
```

File: UPGRADE.md [hunk 2]
```diff
@@ -33,7 +33,117 @@ ALTER TABLE test_example_dimension_contents DROP dimension_id;
...
+
+<details>
+<summary>Doctrine Migration Example</summary>
```
Comment (niklasnatter) [inline UPGRADE.md]:
```suggestion
<summary>DoctrineMigrationBundle Example</summary>
```

File: UPGRADE.md [hunk 3]
```diff
@@ -33,7 +33,117 @@ ALTER TABLE test_example_dimension_contents DROP dimension_id;
...
+     * @return array<array<string, string>>
+     */
+    private function tableData(): array
+    {
+        return [
+            [
+                // TODO replace the following values with the ones by your table
+                'tableName' => 'my_entity_dimension_content',
+                'foreignKey' => 'FK_61A94F1277428AD',
+                'indexName' => 'IDX_61A94F1277428AD',
```
Comment (niklasnatter) [inline UPGRADE.md]:
```suggestion
                'tableName' => 'my_entity_dimension_content', # name of the table of your DimensionContent entity
                'foreignKey' => 'FK_61A94F1277428AD', # dimension foreign key on the DimensionContent table
                'indexName' => 'IDX_61A94F1277428AD', # dimension index on the DimensionContent table
```

**Comment by alexander-schranz** [comment]
@nnatter Thank you :)

**Pull request merged**